src/ucp/api/ucp.h

```diff
@@ -1142,9 +1142,8 @@ ucs_status_t ucp_put_nbi(ucp_ep_h ep, const void *buffer, size_t length,
  * address @a remote_addr and the @ref ucp_rkey_h "memory handle" @a rkey in
  * the local contiguous memory region described by @a buffer address.  The
  * routine returns when remote data is loaded and stored under the local address
- * @e buffer. If the operation is completed immediately the routine return UCS_OK,
- * otherwise UCS_INPROGRESS or an error is returned to user.
-
```

Comment:
@yosefe - I think we still want to distinguish between inplace completion and in-progress even for implicit non-blocking.